### DISCUSSION TITLE
How do I get the Yolo and Gitty Up medals

### DISCUSSION THREAD

**Discussion by chandran-jr** [community/community#38181] [Other Feature Feedback, Questions, & Ideas] [answered]
I did what was suggested.. merge a pull req within 5 minutes and did it without code review.. but I'm not getting it. Welp

**Comment by david-engelmann** [comment]
Maybe try them separately ie. merge a pull req you reviewed within 5 minutes. Then do a second PR without a code review

**Reply by chandran-jr** [reply]
Any idea how long does it take to reflect on our profile?

**Comment by SilasRodrigues19** [comment, marked as answer]
@chandran-jr To get the Quick Draw (Gitty Up) achievement, simply open an issue and close it in less than 5 minutes and to get the Yolo you need merge a PR without code review.

**Reply by SilasRodrigues19** [reply]
In this repo there is a complete list with all achievements and their tiers [https://github.com/drknzz/GitHub-Achievements](https://github.com/drknzz/GitHub-Achievements)

**Reply by chandran-jr** [reply]
Any idea how long does it take to reflect on our profile?

**Reply by SilasRodrigues19** [reply]
Appears almost instantly

**Reply by ayushkrrana** [reply]
> @chandran-jr To get the Quick Draw (Gitty Up) achievement, simply open an issue and close it in less than 5 minutes and to get the Yolo you need merge a PR without code review.

I have close the issue within 5 minutes , but still it did not show gitty up badge on my profile @SilasRodrigues19

**Reply by AlvarezMar** [reply]
I've tried EVERYTHING and I can't seem to get it. :(

**Reply by yuanx749** [reply]
Does merging a pull request not count as Quickdraw?

**Comment by KevinRJF** [comment]
is it possible to get the YOLO badge without a reviewer account?

**Reply by prasadcode58** [reply]
No, its not

**Reply by Chamal1120** [reply]
Not possible afaik. 
Best way is to do a simple project with a close friend. So you can share parts merge the PR.
